tests/network/localnet/liblocalnet.py

```diff
@@ -1,15 +1,39 @@
 from libs.net import netattachdef
-from libs.net.vmspec import add_network_interface, add_volume_disk
+from libs.net.traffic_generator import Client, Server
+from libs.net.vmspec import IP_ADDRESS, add_network_interface, add_volume_disk, lookup_iface_status
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from libs.vm.spec import CloudInitNoCloud, Interface, Metadata, Multus, Network
 from libs.vm.vm import BaseVirtualMachine, cloudinitdisk_storage
 from tests.network.libs import cloudinit
 
-NETWORK_NAME = "localnet-network"
+_IPERF_SERVER_PORT = 5201
 
 
-def localnet_vm(namespace: str, name: str, network: str, cidr: str) -> BaseVirtualMachine:
+def run_vms(
+    *vms: BaseVirtualMachine,
+) -> tuple[BaseVirtualMachine, ...]:
+    for vm in vms:
+        vm.start()
+    for vm in vms:
+        vm.wait_for_ready_status(status=True)
+        vm.wait_for_agent_connected()
+    return vms
+
+
+def create_traffic_server(vm: BaseVirtualMachine) -> Server:
+    return Server(vm=vm, port=_IPERF_SERVER_PORT)
+
+
+def create_traffic_client(vm_server: BaseVirtualMachine, vm_client: BaseVirtualMachine, network_name: str) -> Client:
```

Comment:
nitpick but wouldn't naming `server_vm` and `client_vm` will be more accurate?

Reply:
What exactly you changed?

Reply:
Not sure I understand what's changed. I was referring to the arg names.
So I think that `server_vm` is more clear than `vm_server` because the function gets a VM and not a server.
(as mentioned above, this is a nitpick, please decide)

Reply:
This was the original signature:
`def create_traffic_client(vms: tuple[BaseVirtualMachine, BaseVirtualMachine], network_name: str) -> Client:`
I'll rename to start with the function and end with vm.